page-changelog.php

```diff
@@ -15,6 +15,108 @@
+         <li>Fixed double escaping in full page error messages - <a  href="https://github.com/owncloud/core/issues/29304">#29304</a></li>
+         <li>Updated davclient.js which fixes issue whenever an app extends Array prototype - <a  href="https://github.com/owncloud/core/issues/29305">#29305</a></li>
+         <li>Fix OCS apps API to correctly include attributes into generated XML - <a  href="https://github.com/owncloud/core/issues/29303">#29303</a></li>
+         <li>Make enum type mapping work with migrations - <a  href="https://github.com/owncloud/core/issues/29268">#29268</a></li>
+         <li>Handle invalid storage when getting storage root id - <a  href="https://github.com/owncloud/core/issues/29278">#29278</a></li>
+         <li>Fix storing/retrieval for dav properties of non files - <a  href="https://github.com/owncloud/core/issues/29273">#29273</a></li>
+         <li>Remove double quotes from boolean values in status.php output - <a  href="https://github.com/owncloud/core/issues/29271">#29271</a></li>
+         <li>Tidy code in DAV related classes - <a  href="https://github.com/owncloud/core/issues/29272">#29272</a></li>
+         <li>Fix the missing argument to DecryptAll - <a  href="https://github.com/owncloud/core/issues/29371">#29371</a></li>
+         <li>Skip copying skeleton files if skeleton dir is not accessible - <a  href="https://github.com/owncloud/core/issues/29379">#29379</a></li>
+         <li>Use chunked DB query when preloading directory content for DAV properties - <a  href="https://github.com/owncloud/core/issues/29416">#29416</a></li>
+         <li>Fix failure when checking integrity signature for non-existing files - <a  href="https://github.com/owncloud/core/issues/29433">#29433</a></li>
+         <li>Prevent uploading of part files through WebDav - <a  href="https://github.com/owncloud/core/issues/29432">#29432</a></li>
+         <li>Only trigger "changeUser" event if account object really changed - <a  href="https://github.com/owncloud/core/issues/29429">#29429</a></li>
+         <li>Only load app type once in app manager classes - <a  href="https://github.com/owncloud/core/issues/29428">#29428</a></li>
+         <li>Use efficient startsWith implementation in server container - <a  href="https://github.com/owncloud/core/issues/29427">#29427</a></li>
+         <li>Fix race condition in browser when uploading folder tree - <a  href="https://github.com/owncloud/core/issues/29435">#29435</a></li>
+         <li>Disable nginx buffering for file downloads to avoid huge memory usage in some scenarios - <a  href="https://github.com/owncloud/core/issues/29403">#29403</a></li>
+         <li>Fix many issues related to session removal - <a  href="https://github.com/owncloud/core/issues/28879">#28879</a></li>
+         <li>Fix SMB to better detect when overwriting through rename - <a  href="https://github.com/owncloud/core/issues/29564">#29564</a><br></li>
```

Comment:
seems the magic script decided to add a `<br>` at the end of the last line. not sure why.